modules/programs/zellij.nix

```diff
@@ -49,9 +49,28 @@ in {
       default = false;
     };
 
-    enableFishIntegration = mkEnableOption "Fish integration" // {
+    enableFishIntegration = mkEnableOption
+      "Fish integration (enables both enableFishAutoStart and enableFishCompletions)"
+      // {
+        default = false;
+      };
+    enableFishCompletions = mkEnableOption "load zellij completions" // {
       default = false;
```

Comment:
These completion options should be true by default. This is true for all shells afaik.

Reply:
Good call. changed :)